mmv1/templates/terraform/examples/redis_cluster_ha.tf.erb

```diff
@@ -9,6 +9,9 @@ resource "google_redis_cluster" "<%= ctx[:primary_resource_id] %>" {
   node_type = "REDIS_SHARED_CORE_NANO"
   transit_encryption_mode = "TRANSIT_ENCRYPTION_MODE_DISABLED"
   authorization_mode = "AUTH_MODE_DISABLED"
+  redis_configs {
```

Comment:
Currently this syntax is treating this field like a block, but as the field is a map of arbitrary string key:value pairs it needs this `=`, like you've done elsewhere in your PR. Making this change will address the current problem in the TestAccRedisCluster_redisClusterHaExample test.

I think this might be the cause of the missing test detector saying the field isn't being tested correctly. We can see what the tool says once a change is pushed 😁

Reply:
yeah good catch! I think this is the problem. Let me fix it now.